### PR TITLE
Promote rooms feature to GA

### PR DESCRIPTION
**Pull request by mgamis-msft**
# What
Promote rooms feature to GA
 
# Why
To get it ready for stable release in December. See if snapshots need to be updated

# How Tested
Ran local sample with smoke tests.

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/react-composites/tests/browser/snapshots/stable/tests/browser/call/hermetic/PageState.test.ts-snapshots/permission-denied-to-room-page-Mobile-Android-Portrait-linux.png
Comment (dmceachernmsft) [inline packages/react-composites/tests/browser/snapshots/stable/tests/browser/call/hermetic/PageState.test.ts-snapshots/permission-denied-to-room-page-Mobile-Android-Portrait-linux.png:1]: Something here seems a little out of balance. should there be more padding on the left?

File: packages/react-composites/tests/browser/snapshots/stable/tests/browser/call/hermetic/RoomsCallScreen.test.ts-snapshots/rooms-call-screen-consumer-click-more-button-Mobile-Android-Landscape-linux.png
Comment (dmceachernmsft) [inline packages/react-composites/tests/browser/snapshots/stable/tests/browser/call/hermetic/RoomsCallScreen.test.ts-snapshots/rooms-call-screen-consumer-click-more-button-Mobile-Android-Landscape-linux.png:1]: Why are these not in the bottom sheet drawer? do we want to  not use the side pane here?

Comment (mgamis-msft) [reply packages/react-composites/tests/browser/snapshots/stable/tests/browser/call/hermetic/RoomsCallScreen.test.ts-snapshots/rooms-call-screen-consumer-click-more-button-Mobile-Android-Landscape-linux.png:1]: The rooms call screen snapshots are using the old control bar which are still using the flyout. I'm potentially working on stabilizing the people button for the old control bar but that is in a separate PR

File: packages/react-composites/tests/browser/snapshots/stable/tests/browser/call/hermetic/PageState.test.ts-snapshots/permission-denied-to-room-page-Mobile-Android-Portrait-linux.png
Comment (mgamis-msft) [reply packages/react-composites/tests/browser/snapshots/stable/tests/browser/call/hermetic/PageState.test.ts-snapshots/permission-denied-to-room-page-Mobile-Android-Portrait-linux.png:1]: Yeah that is weird will see what's up here.

**Review by dmceachernmsft — APPROVED** [review]

File: packages/communication-react/review/stable/communication-react.api.md [hunk 1]
```diff
@@ -1949,13 +1965,17 @@ export const DEFAULT_COMPOSITE_ICONS: {
     NoticePageJoinCallFailedDueToNoNetwork?: JSX.Element | undefined;
     NoticePageLeftCall?: JSX.Element | undefined;
     NoticePageRemovedFromCall?: JSX.Element | undefined;
-    OptionsCamera: JSX.Element;
-    OptionsMic: JSX.Element;
-    OptionsSpeaker: JSX.Element;
-    ParticipantItemMicOff: JSX.Element;
-    ParticipantItemScreenShareStart: JSX.Element;
-    VideoTileMicOff: JSX.Element;
+    NoticePageNotInvitedToRoom?: JSX.Element | undefined;
```
Comment (JamesBurnside) [inline packages/communication-react/review/stable/communication-react.api.md:1968]: Might be the api extractor, but these should just have the `?` and not have the `| undefined`

**Review by JamesBurnside — APPROVED** [review]

File: packages/communication-react/review/stable/communication-react.api.md (hunk 1 above)
Comment (mgamis-msft) [reply packages/communication-react/review/stable/communication-react.api.md:1968]: I think it is the api extractor but at least it is consistent.

**Pull request merged**